Resources/doc/versioning.rst

```diff
@@ -0,0 +1,180 @@
+API versioning
+==============
+Why version your API ?
+----------------------
+If you want to introduce breaking changes in your API, you should think about version your API. Indeed, if you introduce them on your current API, your users won't necessary upgrade their application directly and their applications won't work.
+
+How to version your API ?
+-------------------------
+There are several ways, none is standard :
+    - Use a query parameter ``/users?version=v1``
+    - Use a custom mime-type with an ``Accept`` header ``Accept: application/json; version=1.0``
+    - Use a custom header ``X-Accept-Version: v1``
+
+The ``FOSRestBundle`` allows you to use several of them at the same time or to choose one of them.
+
+URI API versioning
```

Comment:
i think this is now a bit confusing .. I guess I would still mention URI versioning in the list above.